src/App.js

```diff
@@ -3,13 +3,17 @@ import Home from "./pages/Home";
 import Rules from "./pages/Rules";
 import Result from "./pages/Result";
 import Quiz from "./pages/Quiz";
+import { Routes, Route } from "react-router-dom";
+
 function App() {
   return (
     <div className="App">
-      <Home />
-      <Rules />
-      <Result />
-      <Quiz />
+      <Routes>
```

Comment:
You can put routes into a separate file and import that component here for cleaner code

Reply:
Thanks for the sugession. Will implement it.